chapters/6-analysis.tex

```diff
@@ -106,84 +105,82 @@ \subsection{Fitting construction into the data workflow}
 In such cases, the main source of unexpected results are unmatched observations.
 Fortunately, it is straightforward to avoid mistakes in this setting:
 before writing code to combine the datasets,
 write pseudocode to understand which observations you expect to be
 matched or not, and why.
-When possible, determine exactly which and how many 
+When possible, determine exactly which and how many
 matched and unmatched observations should result from the merge.
-For example, first that were created in the second year should not be
-present in the first year's data, 
-and firms that closed during the first year should not be observed in the second year's data. 
+For example, firms that were created in the second year should not be
+present in the first year's data,
+and firms that closed during the first year should not be observed in the second year's data.
 Then think carefully about whether you want to keep matched and unmatched
-observations, or only specific matching outcomes,
- for example, to create a balanced panel,
-and add that to your pseudocode as well.
-Now you can merge the datasets, 
+observations, or only specific matching outcomes (e.g. to create a balanced panel).
+Add that to your pseudocode as well.
+Now you can merge the datasets,
 and test that the outcome matches your expectations.
-Add comments to explain any exceptions, 
+Add comments to explain any exceptions,
 and make it so the code will return an error in case unexpected results show up in future runs.
 This approach can be extended to other types of merges between two related datasets.
 % to-do % ----------------------------------------------------------------------------
 % See XXX for an example of a Stata do-file that implements these practices -- Cami's code from TUP
 % ------------------------------------------------------------------------------------
 
 % Fuzzy matching
-Even when matching data sets with the same unit of observations, however,
-more complex cases may arise.
-Datasets at the same unit of observation may not use a consistent identifier.
-One common example is for dataset identifiers to be come in the form of names, 
-which are often spelled differently in different data sources. 
-In these cases, you will need to do a \textbf{fuzzy match}, 
-to link observations that have similar identifiers. 
+Even when merging data sets with the same unit of observations, however,
+complex cases may arise.
+Datasets with the same unit of observation may not have a consistent identifying variable.
+One common example is for dataset identifiers to be in the form of names,
+which are often spelled differently in different data sources.
+In these cases, you will need to do a \textbf{fuzzy match},
+to link observations that have similar identifiers.
 You will need to analyze the merging patterns extensively,
 understand what units are present in one dataset but not the other,
 and investigate unmatched observations to resolve imperfect matching.
```

Comment:
Discussed in chapter 3 in relation to master datasets.

Reply:
Done in 06f9055